neuralteleportation/experiments/teleport_training.py

```diff
@@ -18,14 +18,16 @@
 from neuralteleportation.training.teleport import optim as teleport_optim
 from neuralteleportation.training.teleport.optim import OptimalTeleportationTrainingConfig
 from neuralteleportation.training.teleport.pseudo import PseudoTeleportationTrainingConfig
+from neuralteleportation.training.teleport.pseudo import DistributionTeleportationTrainingConfig
 from neuralteleportation.training.teleport.random import RandomTeleportationTrainingConfig
 from neuralteleportation.utils.itertools import dict_values_product
 from neuralteleportation.utils.logger import DiskLogger, MultiLogger, CometLogger
 
 __training_configs__ = {"no_teleport": TrainingConfig,
                         "random": RandomTeleportationTrainingConfig,
                         "optim": OptimalTeleportationTrainingConfig,
-                        "pseudo": PseudoTeleportationTrainingConfig}
+                        "pseudo": PseudoTeleportationTrainingConfig,
+                        "same_distr": DistributionTeleportationTrainingConfig}
```

Comment:
For this file, I only added these two lines and changed the formatting because of PEP8 issues.

Do you still think I should revert the formatting?

Reply:
I'd still revert the format. It is generally a good idea to follow PEP8 recommendations, but I prefer readability over enforcing those rules. If we want to have a more standard formatting of the code base, we should look at using the code formatter [black](https://black.readthedocs.io/en/stable/), instead of trying to manually adjusting the code.